manual/python/index.md

```diff
@@ -66,9 +66,95 @@ caption="Example of graphics generated with PyROOT."
 
 ## Loading user libraries & jitting
 
-*Content is coming soon!*
+PyROOT allows to use *any C++ library* from Python, not only ROOT. This is possible thanks to the automatic and dynamic bindings between Python and C++ that PyROOT provides: without any prior generation of wrappers, at execution time, PyROOT can load and call into C++ code. The following sub-sections cover different supported scenarios for dynamic C++ code invocation.
+
+### Jitting strings with code
+
+ROOT comes with a C++ interpreter to which we can request to process a given piece of C++ code. Sometimes, if such code is short (e.g. the definition of a small function or class) or for rapid exploration/debugging, it is enough the place the C++ code in a Python string and give it to the interpreter. The code will be just-in-time compiled (jitted) and immediately available for invocation, as shown in the example below.
+
+```python
+import ROOT
+
+# Write some C++ code in a string
+cpp_code = """
+// Function definition
+int f(int i) { return i*i; }
+
+// Class definition
+class A {
+public:
+    A() { cout << "Hello PyROOT!" << endl; }
+};
+"""
+
+# Inject the code in the ROOT interpreter
+ROOT.gInterpreter.ProcessLine(cpp_code)
+
+# We find all the C++ entities in Python, right away!
+a = ROOT.A()   # this prints Hello PyROOT!
```

Comment:
Probably `this calls the constructor of class A and prints Hello PyROOT!`? To make very clear what happens. I can imagine that not everyone sees right away that here a construction happens.

Reply:
I made a more verbose explanation of what happens in the text before the example, to not make the comment too long in the code.